pom.xml

```diff
@@ -119,6 +119,17 @@
   </dependencies>
 
   <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <!-- review with next pom or next m-resources-p -->
```

Comment:
Please explicitly mention https://issues.apache.org/jira/browse/MSHARED-1113 to ease reviewing